gprofiler/profilers/perf.py

```diff
@@ -245,6 +247,14 @@ def __init__(
         self._node_processes_attached: List[Process] = []
         self._perf_memory_restart = perf_memory_restart
         switch_timeout_s = duration * 3  # allow gprofiler to be delayed up to 3 intervals before timing out.
+        extra_args = []
+        if is_profiler_disabled(perf_mode):
```

Comment:
Huh? If perf mode is none/disabled then we don't want to run any perf at all, what is this check for?

Reply:
yea it should have been ```suggestion
        if not is_profiler_disabled(perf_mode):
```

Reply:
63b1be26ae67e5177b94768c73485637fb78f9f0